### PR TITLE
updated electron

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
## Summary

Updated electron to 4.0.1

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Issue

https://github.com/bbc/bbc-a11y/issues/274

### CODE REVIEW

**Comment by oluoluoxenfree** [comment]
I had an error locally which was the (ongoing?) `--help` indentation thing, but my errors on CI are completely different; anyone have any idea why that would be?

**Comment by ChrisBAshton** [comment]
Might want to update to 2.0.8 as per #274, rather than 4.0.1 which is two major versions ahead (so likely to have breaking API changes).

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # v2.2.6
 
+- Upgraded Electron to v2.0.8
+
+# v2.2.6
```
Comment (ChrisBAshton) [inline HISTORY.md]: This is the same version as on line 1 - should line 1 be updated to 2.2.7?

**Comment by andymsuk** [comment]
Version bumping is done as part of the release process as it needs to be done elsewhere too, so can you remove the version change in package.json and remove change in HISTORY?

**Comment by EmmaJP** [comment]
> Version bumping is done as part of the release process as it needs to be done elsewhere too, so can you remove the version change in package.json and remove change in HISTORY?

Did the bbc-a11y version change get removed yet from this PR? Asking because I'm still seeing it in the 'Files changed'.

**Comment by oluoluoxenfree** [comment]
Not yet, I can do it this evening.

**Comment by oluoluoxenfree** [comment]
Is there anything else I need to do for this? @ChrisBAshton @andymsuk @EmmaJP

**Comment by andymsuk** [comment]
The update to the HISTORY file should be removed as it's done as part of the release process.

other than that, it's ready to merge and release - just need to find time to do it. Maybe @ChrisBAshton is more free than me?

**Comment by ChrisBAshton** [comment]
I'm not, unfortunately!

**Comment by andymsuk** [comment]
Hi @oluoluoxenfree 
Apologies, I'd forgotten about this PR and now there's conflicts. 
If you can resolve the conflicts, I can look to get this merged.

**Comment by oluoluoxenfree** [comment]
I think I might close this one and start a new PR with the updated master branch; there are a lot of changes in the package,.json

**Pull request closed without merging**